variables.tf

```
@@ -58,14 +58,14 @@ variable "db_port" {
 
 variable "admin_user" {
   type        = string
-  default     = "admin"
-  description = "(Required unless a snapshot_identifier is provided) Username for the master DB user"
+  default     = ""
```

```suggestion
  default     = "admin"
```
To preserve backwards compatibility, we do not want to change default values.